extension/src/format/sexpression.ts

```diff
@@ -325,29 +325,75 @@ export class LispContainer extends LispAtom {
 
 
 
-enum  LongListFmts{
+enum LongListFmts {
     kSingleColumn,
     kWideStyleSingleCol,
     kFitToMargin
-  }
-  
-  let gMaxLineChars = 80;
-  let gIndentSpaces = 2;
-  let gClosedParenInSameLine = true;
-  let gLongListFormatAsSingleColumn = LongListFmts.kFitToMargin;
-  let gHasSetLongListFormat = false;
-  export function longListFormatAsSingleColum() {
```

Comment:
Let's not format the entire file in a PR that contains real code changes. It takes extra time to review things that are not really changed.
If the formatting is wrong, we can submit the test case change first, then create another PR which is purely for code format.